### DISCUSSION TITLE
Horizontal scrolling layouts

### DISCUSSION THREAD

**Discussion by wheelie33** [xamarin/Xamarin.Forms#13129] [Q&A]
![Drawing](https://user-images.githubusercontent.com/17298183/102019186-9b201a00-3d37-11eb-9978-2e0121c69e99.jpeg)

I've looked for HOURS to no avail to find a layout that allows me the screenshot functionality.   I'd prefer not to have 1000's of custom lines of code or wonky, outdated nuget packages, or $1,000 custom plugins if possible.

**Comment by davidortinau** [comment]
Where is the viewport, the center column?

Perhaps a CarouselView would suffice? 

https://github.com/davidortinau/CarouselGallery/blob/master/CarouselGallery/CarouselGallery/Views/MintPage.xaml


![Simulator Screen Shot - iPhone 12 - 2020-12-13 at 21 14 51](https://user-images.githubusercontent.com/41873/102036368-4789ec80-3d88-11eb-9c13-645acbe3dcef.png)

If you have content that should remain in the viewport while other content slides behind it, then a Grid layout allows you that z-ordering.
